packages/amazonq/test/e2e/lsp/lspInstaller.test.ts

```diff
@@ -117,6 +121,86 @@ describe('AmazonQLSPInstaller', () => {
             assert.ok(fallback.assetDirectory.startsWith(tempDir))
             assert.deepStrictEqual(fallback.location, 'fallback')
             assert.ok(semver.satisfies(fallback.version, supportedLspServerVersions))
+
+            // Exclude version numbers so that this test doesn't have to be updated on each update.
+            // Fetching manifest fails locally, but passes in CI. Therefore, this test will fail locally.
```

Comment:
Can you explain more about this? The test will fail locally?

Reply:
When running locally, the manifest fetch fails on both the first and second try. However, when it runs in CI the first try succeeds and the second one does not. Not entirely sure why, but it appears to be consistent across the 3 times I've run it so far. How it ultimately resolves the manifest impacts which telemetry metrics are emitted so the metrics emitted when run locally differ from that in CI. Therefore, I adjusted the test so that it the expected behavior is the behavior in CI.

Reply:
After diving deeper on this I discovered the root issue, and it should be fixed now. To summarize, the globalState is consistent in CI, but not local since we don't overwrite it on e2e/integ tests. This means that if I've already downloaded the LSP locally, the manifest etag is in globalState. If this eTag is up to date, the remote fetch fails. 

I fixed it by temporarily clearing the manifest storage in globalState for these tests, and then restoring it afterwards.